examples/camera-controller/webrtc_manager/WebRTCManager.cpp

```diff
@@ -209,6 +198,9 @@ void WebRTCManager::HandleCommandResponse(const ConcreteCommandPath & path, TLV:
 {
     ChipLogProgress(NotSpecified, "Command Response received.");
 
+    // TODO: Upon receiving, the client SHALL create a new WebRTCSessionStruct populated with the values from this command, along
+    // with the accessing Peer Node ID and Local Fabric Index entries stored in the Secure Session Context, as the PeerNodeID and
+    // FabricIndex values, and store in the Requestor clusters CurrentSessions.
     if (path.mClusterId == Clusters::WebRTCTransportProvider::Id &&
         path.mCommandId == Clusters::WebRTCTransportProvider::Commands::ProvideOfferResponse::Id)
     {
```

Comment:
The HandleProvideOfferResponse would be creating the Session struct. From the controller side, I am guessing accessing the PeerEndPointID will not be that difficult. Would that be all handled inside the HandleProvideOfferResponse() command or does it need to be passed in?

Reply:
https://github.com/project-chip/connectedhomeip/pull/38253